src/main/java/id/shivelight/paper/unexpectedspawn/UnexpectedSpawn.java

```diff
@@ -28,9 +28,11 @@
 
     @Override
     public void onEnable() {
-        config = new ConfigAccessor(this, "config.yml");
-        config.getConfig().options().copyDefaults(true);
-        config.saveConfig();
+        this.config = new ConfigAccessor(this, "config.yml");
+        this.config.getConfig().options().copyDefaults(true);
+//        config.saveConfig();
+//        used this so the comment in config.yml is preserved.
+        this.config.saveDefaultConfig();
```

Comment:
Just adding future concerns; if the config structure is changed, it will not merge the existing configuration (new keys will be missing). Looks good for now.

Reply:
Yes, user need to remove any previous build or config files.